lib/Dialect/Triton/Transforms/Combine.td

```diff
@@ -12,7 +12,7 @@ include "mlir/IR/PatternBase.td"
 // AddIOp(d, DotOp(a, b, c)) and c==0 => DotOp(a, b, d)
 // AddFOp(d, DotOp(a, b, c)) and c==0 => DotOp(a, b, d)
 def CombineDotAddIPattern : Pat<
-        (Arith_AddIOp $d, (TT_DotOp:$res $a, $b, $c, $allowTF32, $maxNumImpreciseAcc)),
+        (Arith_AddIOp $d, (TT_DotOp:$res $a, $b, $c, $allowTF32, $maxNumImpreciseAcc), $overflow),
```

Comment:
Just to check, did Arith_AddIOp lose the overflow flag?

Reply:
No, it gained one in https://github.com/llvm/llvm-project/pull/78376, hence we have to pattern-match it.